SUUpdateAlert.m

```diff
@@ -280,9 +280,15 @@ - (void)awakeFromNib
 		[self displayReleaseNotes];
 	}
 
-	if (![self allowSkipUpdates])
+	if ( [updateItem isCriticalUpdate] || [updateItem isMandatoryUpdate] )
 	{
-		[skipButton setHidden: YES];
+        if( ![self allowSkipUpdates] ) {
```

Comment:
Q. Should we allow to skip updates even when it is a critical (or mandatory) update? Seems kinda contradictory....

Reply:
(from discussion with Chris)

so, here is Sparkle v2

> if (_updateItem.criticalUpdate && !_updateItem.majorUpgrade) {
>     _skipButton.hidden = YES;
>     _laterButton.hidden = YES;
>   }

as for window close button -  I didnt check but applied what seems reasonable. I have a gut feeling they implemented that anyway somewhere
Side notes…. I will remove manadatory then as too many questions appears around it.
Also, to me, the extension for allowSkipUpdate is not good and does not follow Sparkle standard. and this is what will break us more. especially since the value reads from .plist first.
so I will remove that also